packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/UpgradeNowSilver/index.tsx

```diff
@@ -59,7 +60,16 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
 })
 
 const mapStateToProps = (state: RootState) => ({
-  data: getData(state)
+  data: getData(state),
+  userData: selectors.modules.profile.getUserData(state).getOrElse({
```

Comment:
Do you need `userData` selector here and the same code in the selectors.ts file?